lib/magma/commands.rb

```diff
@@ -2,6 +2,42 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    def execute(file_name)
+      file = File.open(file_name)
+      file_data = JSON.parse(JSON.parse(file.read))
+
+      db = Magma.instance.db
+
+      models = file_data["models"]
+
+      models.keys.each do |model|
+        model_name = models[model]["template"]["name"]
+        attributes = models[model]["template"]["attributes"]
+
+        attributes.keys.each do |att|
+          attribute = attributes[att]
+
+          db[:attributes].insert(
+            project_name: 'i dont know where to find the project name',
```

Comment:
Hmm - doesn't return in the payload from /retrieve at the moment, so I guess it would be the first argument to the `load_project` method?